merge_requests.go

```diff
@@ -105,6 +105,7 @@ type MergeRequest struct {
 	HasConflicts                bool `json:"has_conflicts"`
 	BlockingDiscussionsResolved bool `json:"blocking_discussions_resolved"`
 	Overflow                    bool `json:"overflow"`
+	AllowCollaboration          bool `json:"allow_collaboration"`
```

Comment:
Maybe add `allow_maintainer_to_push` as well? And please order them following the docs (so these should be above the `WebURL` field.

Reply:
> Maybe add allow_maintainer_to_push as well?

`allow_maintainer_to_push` is deprecated in favour of `allow_collaboration` field so I don't think it should be added

Reply:
> And please order them following the docs (so these should be above the WebURL field.

Fixed that

Reply:
Ah, good point. Didn't know that 👍🏻